mocked_servers/https/README.md

```diff
@@ -0,0 +1,5 @@
+# HTTPS Server
```

Comment:
Please add a comment saying open ssl must be installed without sudo requirement for this to work. At least I think you will need to install open ssl for this to work.